app/hotels/src/map/allHotels/MapView.js

```diff
@@ -175,7 +165,15 @@ export class Map extends React.Component<Props, State> {
       return;
     }
 
-    this.map.animateToCoordinate(coordinate, duration);
+    this.map.animateCamera(
```

Comment:
changes in API, I see.

Reply:
Yes, animateToCoordinate has been deprecated